TCAT/Controllers/InformationViewController.swift

```diff
@@ -42,138 +43,137 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
 
     ]
 
+    private let descriptionLabelSize = CGSize(width: 167, height: 34)
+    private let descriptionLabelTopOffset: CGFloat = 12
+    private let headerWidth = UIScreen.main.bounds.width
+    /** Represents the total height of the headerView given the width of the phone screen. Most of the variables
+     used are static except for the tcat image height, which is dependent on the width of the user's screen.
+     The extra 37 is used for bottom inset padding for the headerView, but I felt it would be unnecessary to
+     create a variable for it */
+    private var headerHeight: CGFloat {
+        return tcatImageTopOffset + tcatImageSize.height + titleLabelTopOffset + titleLabelSize.height + descriptionLabelTopOffset + descriptionLabelSize.height + 37
+    }
+    private var tcatImageSize: CGSize { return CGSize(width: headerWidth-80, height: (headerWidth - 80) * 0.4) }
```

Comment:
should be `headerWidth - 80` and not `headerWidth-80`, @kevinchan159 thoughts on making this multi-line? I'm personally not a fan of inlining `return` statements that have any sort of logic

Reply:
yeah i think multiline is prob better